mix.exs

```diff
@@ -101,7 +101,8 @@ defmodule Crawler.MixProject do
         "cmd npm run eslint.fix --prefix assets",
         "prettier.fix",
         "deps.clean --unlock --unused",
-        "format"
+        "format",
+        "gettext.extract --merge --no-fuzzy"
```

Comment:
Nice 👍 we had the `"gettext.extract-and-merge": ["gettext.extract --merge --no-fuzzy"],` above but I think it is better to have it in the `codebase.fix` here 👍